scripts/mkcloud

```diff
@@ -1360,6 +1385,18 @@ function sanity_checks
         complain 113 "Please increase number of nodes for this setup, minimal nodenumber=3"
     fi
 
+    if [[ $want_ceph = 1 ]] && [[ $want_external_ceph -gt 0 ]] ; then
+        complain 118 "\$want_ceph and \$want_external_ceph are mutully exclusive. Please choose one or the other"
+    fi
+
+    if [[ $want_external_ceph -gt 0 ]] && [[ $(nodes number normal) -lt ${external_ceph_nodes} ]] ; then
+        complain 119 "Please increase number of nodes for this setup, minimal nodenumber=$external_ceph_nodes"
+    fi
```

Comment:
I believe this check is wrong. 
If nodenumber is the amount of nodes used for cloud only. external_ceph_nodes are added extra.
For example nodenumber=3 and external_ceph_nodes=3 will create 6 nodes in total.

So i should be able to create configurations where nodenumber <= external_ceph_nodes.

Reply:
yeah this is wrong. originally the step was to be called as part of normal cloud build. Now its expected to be called with lonelynodes. I removed all other instaces of external_ceph_nodes.. and forgot these.

Nice spot.